components/expenses/ExpenseForm.js

```diff
@@ -367,6 +371,14 @@ const ExpenseFormBody = ({
     }
   }, [formPersister, dirty, values]);
 
+  const handleClearPayeeStep = () => {
+    setShowResetModal(true);
+  };
+
+  const resetDefaultStep = () => {
+    setStep(EXPENSE_FORM_STEPS.PAYEE);
+  };
```

Comment:
Minor suggestion: I am wondering the usefulness of these two functions, they are both used only ones and thus I would say maybe we can just use these functions inline within the `ExpenseFormPayeeStep`;

```
...
handleClearPayeeStep={() => setShowResetModal(true)}
...
```